diff --git a/Cargo.toml b/Cargo.toml
index 25b3240..c81725c 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -86,8 +86,18 @@ assets = [
 [profile.release]
 lto = true

-##### uncomment to test the newest librespot git version
-# [patch.crates-io]
+##### local librespot with manual track-switch ramp
+[patch.crates-io]
+librespot-audio = { path = "../librespot/audio" }
+librespot-playback = { path = "../librespot/playback" }
+librespot-core = { path = "../librespot/core" }
+librespot-discovery = { path = "../librespot/discovery" }
+librespot-connect = { path = "../librespot/connect" }
+librespot-metadata = { path = "../librespot/metadata" }
+librespot-protocol = { path = "../librespot/protocol" }
+librespot-oauth = { path = "../librespot/oauth" }
+
+# uncomment to test the newest librespot git version
 # librespot-audio = { git = "https://github.com/librespot-org/librespot" }
 # librespot-playback = {  git = "https://github.com/librespot-org/librespot" }
 # librespot-core = { git = "https://github.com/librespot-org/librespot" }
